.gitlab-ci.yml

```diff
@@ -24,7 +24,9 @@ variables:
   # this var is changed to "-:staging" when the CI image gets rebuilt
   # read more https://github.com/paritytech/scripts/pull/244
   ALL_CRATES:                      "${PURELY_STD_CRATES} ${ALSO_WASM_CRATES}"
-  DELEGATOR_SUBCONTRACTS:           "accumulator adder subber"
+  DELEGATOR_SUBCONTRACTS:          "accumulator adder subber"
+  # until https://github.com/rust-lang/cargo/issues/5034 has been implemented
```

Comment:
I would make an issue in the ink! repo which mentions this + some more context. Then I'd link _that_ issue here